src/bin/common/solving_loop.ml

```diff
@@ -546,15 +587,13 @@ let main () =
             end
           | None ->
             (* TODO: add the location of the statement. *)
-            Printer.print_smtlib_err "No model produced.";
-            st
+            recoverable_error "No model produced."; st
         else
-          begin
-            (* TODO: add the location of the statement. *)
-            Printer.print_smtlib_err
-              "Model generation disabled (try --produce-models)";
-            st
-          end
+          (* TODO: add the location of the statement. *)
+          let () =
+            recoverable_error "Model generation disabled (try --produce-models)"
+          in
+          st
```

Comment:
```suggestion
          recoverable_error "Model generation disabled (try --produce-models)";
          st
```

Reply:
We are in an ITE! I could use a `begin end` though...